### DISCUSSION TITLE
[Question] Redis stream scaler - lagCount vs activationLagCount

### DISCUSSION THREAD

**Discussion by kkocel** [kedacore/keda#6091] [Q&A / Need Help] [answered]
I tried to understand the difference between `lagCount` and `activationLagCount` by reading https://github.com/kedacore/keda/blob/main/pkg/scalers/redis_streams_scaler.go.
I wonder why there are two values there instead of one. Why there is no single value that indicates when to scale?

cc: @mikelam-us @mikelam-us-aixplain

**Comment by JorTurFer** [comment, marked as answer]
Hello
You can see how the `activationXXX` works here: https://keda.sh/docs/2.15/concepts/scaling-deployments/#activating-and-scaling-thresholds
